connection_common_integration_test.go

```diff
@@ -19,6 +22,16 @@ const SCAN_STATEMENT_ERROR_MSG = "firebolt rows Scan() call failed with %v"
 const VALUES_ARE_NOT_EQUAL_ERROR_MSG = "values are not equal: %v and %v\n"
 const RESULTS_ARE_NOT_EQUAL_ERROR_MSG = "results are not equal "
 
+var longTestValue int
+
+func init() {
+	var err error
+	longTestValue, err = strconv.Atoi(os.Getenv("LONG_TEST_VALUE"))
```

Comment:
Let's have some default in case it's not provided. I believe you might want to run an unrelated test sometimes and you should be able not to provide this env variable